### ISSUE TITLE
Request

### ISSUE THREAD

**Issue by zeynali** [experiencor/keras-yolo2#229]
```
 Multiscale training
mAP Evaluation
```
I really need these options ,so i want to know when exactly these options will be ready?

**Comment by zeynali** [comment]
you said multi-scale training  will be finish over one week.

**Comment by experiencor** [comment]
It's relatively easy with one YOLOv3. But YOLOv2 involves many different backends. It's much more difficult.

**Comment by zeynali** [comment]
so now working multi-scale training for v3 ? and i want to use mobilenet and other basenet for yolo3. 
Do you do this?

**Comment by experiencor** [comment]
@zeynali I cannot promise anything about different backends for YOLO3. My code for just one backend is complicated enough. Still try to find some way to simplify the tensorflow code. Tensorflow is clearly not designed for the loss function of YOLO.

**Comment by zeynali** [comment]
@experiencor , your yolo2 really complete , but just only multi-scale training.
